Plugins/NativeInput/Source/Windows/NativeInputWindows.cpp

```diff
@@ -0,0 +1,9 @@
+#include "..\Shared\NativeInput.h"
+
+namespace Babylon::Plugins
+{
+    bool NativeInput::Impl::HasMouse()
+    {
+        return true;
```

Comment:
I was thinking we would actually use platform specific APIs to determine whether a mouse is present (similar to what Babylon.js does in the browser). For UWP I think this could be determined from `Windows::Devices::Input::MouseCapabilities::MousePresent`.

Reply:
I had a similar thought while looking at this code, but wonder how we'd handle transient devices? For example on HoloLens we support bluetooth mice, but they might not be present at app launch. We'd presumably want to be responsive to runtime changes (though that might also be better to tackle with a separate PR).

Reply:
Yea we should handle that properly but I’m ok with it being in a later change personally.

Reply:
sounds good